src/components/Avatar/AvatarContent.scss

```diff
@@ -3,6 +3,9 @@
 $avatar-icon-large: 28px;
 $avatar-icon-medium: 18px;
 $avatar-icon-small: 12px;
+$font-size-avatar-large: 18px;
```

Comment:
moved out of typography because its not the default sizes which in css variables now